lib/utils.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
 
         // check the type of the value of that option came from the user
         switch (defaultOptions[id].type) {
+          case 'string':
```

Comment:
What's happening here when it is a `string`?

Reply:
`JSON.parse()` can parse non string primitive data types such as boolean, integer etc. For string it fails so treat value as is and don't parse it.

Reply:
@umeshp7  So we didn't have handling for option with type: `string`, this makes sure it's handled and not default case is applied.